acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/transaction/privacy/PrivacyRequestFactory.java

```diff
@@ -364,6 +368,23 @@ public PrivxCreatePrivacyGroupResponse privxCreatePrivacyGroup(
         GetCodeResponse.class);
   }
 
+  public Request<?, EthCall> privCall(
+      final String privacyGroupId,
+      final Contract eventEmitter,
```

Comment:
We should rename this variable to `contract` or something.

Reply:
Done